src/proto/team.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto2";
+
+package TbotsProto;
+
+import "proto/tbots_timestamp_msg.proto";
+import "proto/vision.proto";
+
+message Robot
+{
+    enum RobotCapability
+    {
+        Dribble = 0;
+        Kick    = 1;
+        Chip    = 2;
+        Move    = 3;
+    }
+    required uint32 id                                = 1;
+    required RobotState current_state                 = 2;
+    repeated RobotCapability unavailable_capabilities = 3;
+    required Timestamp timestamp                      = 4;
+    //    RobotConstants_t robot_constants_;
```

Comment:
Commented out properties are ones which our representation of the same type includes, but I decided not to add to the protobuf representation. If we think these properties should be included to the protobuf messages, I can add them, if not, I will remove the comments.

Reply:
you can remove these commented out bits

Reply:
Removed all commented out proto properties